From 8dbcca93eea768496119d448eef365fa74776e71 Mon Sep 17 00:00:00 2001
From: Ryan Lopopolo <rjl@hyperbo.la>
Date: Thu, 4 Jul 2019 19:40:09 -0700
Subject: [PATCH 2/6] Add fallback for Fixnum#size

---
 lib/mspec/guards/platform.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/mspec/guards/platform.rb b/lib/mspec/guards/platform.rb
index ab2c477..e04254f 100644
--- a/lib/mspec/guards/platform.rb
+++ b/lib/mspec/guards/platform.rb
@@ -40,7 +40,7 @@ class PlatformGuard < SpecGuard
     os?(:windows)
   end
 
-  WORD_SIZE = 1.size * 8
+  WORD_SIZE = 1.size * 8 rescue 4 * 8
 
   POINTER_SIZE = begin
     require 'rbconfig/sizeof'
-- 
2.22.0

