From 8f631bb980b16a4c3c8793a81c84498d01d3e827 Mon Sep 17 00:00:00 2001
From: Ryan Lopopolo <rjl@hyperbo.la>
Date: Thu, 4 Jul 2019 19:42:25 -0700
Subject: [PATCH 4/6] Call Kernel#require instead of Kernel#load

---
 lib/mspec/runner/mspec.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/mspec/runner/mspec.rb b/lib/mspec/runner/mspec.rb
index cfc1184..1d1dbc3 100644
--- a/lib/mspec/runner/mspec.rb
+++ b/lib/mspec/runner/mspec.rb
@@ -85,7 +85,7 @@ module MSpec
       setup_env
       store :file, file
       actions :load
-      protect("loading #{file}") { Kernel.load file }
+      protect("loading #{file}") { Kernel.require file }
       actions :unload
     end
   end
-- 
2.22.0

