From 9263cba95fe44e67c14eeeed9b27b5e0f2ab0a89 Mon Sep 17 00:00:00 2001
From: Ryan Lopopolo <rjl@hyperbo.la>
Date: Thu, 4 Jul 2019 19:38:49 -0700
Subject: [PATCH 1/6] Add mruby engine detection to mspec platform guard

---
 lib/mspec/guards/platform.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/mspec/guards/platform.rb b/lib/mspec/guards/platform.rb
index c69401e..ab2c477 100644
--- a/lib/mspec/guards/platform.rb
+++ b/lib/mspec/guards/platform.rb
@@ -6,7 +6,7 @@ class PlatformGuard < SpecGuard
       case name
       when :rubinius
         RUBY_ENGINE.start_with?('rbx')
-      when :ruby, :jruby, :truffleruby, :ironruby, :macruby, :maglev, :topaz, :opal
+      when :ruby, :jruby, :truffleruby, :ironruby, :macruby, :maglev, :topaz, :opal, :mruby
         RUBY_ENGINE.start_with?(name.to_s)
       else
         raise "unknown implementation #{name}"
-- 
2.22.0

