From 70a648d73baa6b4ca61c7d893580882f1bd0595d Mon Sep 17 00:00:00 2001
From: Ryan Lopopolo <rjl@hyperbo.la>
Date: Thu, 4 Jul 2019 19:43:51 -0700
Subject: [PATCH 6/6] Use explicit call to to_int

---
 lib/mspec/utils/version.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/mspec/utils/version.rb b/lib/mspec/utils/version.rb
index 787a76b..8e09686 100644
--- a/lib/mspec/utils/version.rb
+++ b/lib/mspec/utils/version.rb
@@ -42,7 +42,7 @@ class SpecVersion
 
   def <=>(other)
     if other.respond_to? :to_int
-      other = Integer other
+      other = other.to_int
     else
       other = SpecVersion.new(String(other)).to_i
     end
-- 
2.22.0

