commit 169be1c4a8e3be045a66247ae2b097e192f664dc
Author: Ryan Lopopolo <rjl@hyperbo.la>
Date:   Sun Jun 16 17:21:15 2019 +0100

    mruby requires brackets for interpolating instance variables

diff --git a/src/generated/uri/http.rb b/src/generated/uri/http.rb
index 2e2ebcc1..9625995a 100644
--- a/src/generated/uri/http.rb
+++ b/src/generated/uri/http.rb
@@ -78,7 +78,7 @@ module URI
     def request_uri
       return unless @path

-      url = @query ? "#@path?#@query" : @path.dup
+      url = @query ? "#{@path}?#{@query}" : @path.dup
       url.start_with?(?/.freeze) ? url : ?/ + url
     end
   end
